docs/content/operations/dashboard.md

```diff
@@ -5,6 +5,10 @@ See What's Going On
 
 The dashboard is the central place that shows you the current active routes handled by Traefik. 
 
+<aside class="notice">
+   Currently, the dashboard is in a Work In Progress State while beeing reconstructed for v2. Therefore, the dashboard is currently not working
```

Comment:
missing period at the end of the sentence.

Reply:
Fixed :)